recipe/meta.yaml

```diff
@@ -67,7 +59,7 @@ requirements:
     - jom ==1.1.2  # [win]
     - libffi
     - ncurses  # [not win]
-    - openssl
+    - openssl <3.0.0
```

Comment:
@h-vetinari openssl 3.0.0 seems to be the default for 3.8 but not for 3.7. Where can I see the conda-forge patches to `_ssl` to get openssl 3.0.0 working?

Reply:
3.0 is not the default for anything yet; there's 1.1.1 builds for everything, and as far as I can see, also the 3.8 builds here are pulling in 1.1.1.

To answer your questions, here are the conda-forge patches for 3.8 (only one was added in the context of openssl 3): https://github.com/conda-forge/python-feedstock/tree/3.8/recipe

There didn't need to be many, because upstream cpython did basically all of the work for 3.8: https://github.com/python/cpython/commits/3.8/?author=tiran

Reply:
In generally we point people to the `build-locally.py` script as that takes care of some quirks (build in a docker container, set up the macOS SDK, etc.) but `conda build` directly is definitely useful for experienced users to more quickly debug.

@mattip Make sure to revert a646f2a, it'll cause the OpenSSL pinning and migration to fail it's kept.

Reply:
It's at https://github.com/conda-forge/conda-forge.github.io/blob/09bc2d08f0e3ebe583621054a8bda52cf878e75b/docs/_sources/maintainer/updating_pkgs.rst.txt#testing-changes-locally